applications/MappingApplication/tests/basic_mapper_tests.py

```diff
@@ -75,6 +95,15 @@ def test_Map_non_constant_scalar(self):
         self.mapper.Map(KM.PRESSURE, KM.TEMPERATURE)
         mapper_test_case.CheckHistoricalNonUniformValues(self.interface_model_part_destination, KM.TEMPERATURE, GetFilePath(self._GetFileName("map_scalar")))
 
+    def test_Map_non_constant_scalar_expression(self) -> None:
+        SetHistoricalNonUniformSolutionScalar(self.interface_model_part_origin.Nodes, KM.PRESSURE)
+        source_expression = KM.Expression.NodalExpression(self.interface_model_part_origin)
+        KM.Expression.VariableExpressionIO.Read(source_expression, KM.PRESSURE, True)
```

Comment:
```suggestion
        KM.Expression.VariableExpressionIO.Read(origin_expression, KM.PRESSURE, True)
```

please use `origin` and `destination`